### PR TITLE
Resolve VCSWP-19961 (Change AddToConference CallId property type)

### PR DESCRIPTION
**Pull request by amahadaya**
Closes VCSWP-19961

### CODE REVIEW

File: .openapi-generator/FILES [hunk 1]
```diff
@@ -141,7 +141,6 @@ docs/UpdateConferenceRequest.md
 docs/UpdateConferenceRequestStatus.md
 freeclimb.sln
 git_push.sh
-src/freeclimb.Test/Api/DefaultApiTests.cs
```
Comment (katebrune) [inline .openapi-generator/FILES:144]: why is this removed?

Comment (amahadaya) [reply .openapi-generator/FILES:144]: This wasn't auto-generated so it got removed, this keeps a list of all files auto-generated, since there are no changes with the default api tests file, it got removed.

**Review by katebrune — APPROVED** [review]
